src/Game/debugInput.h

```diff
@@ -0,0 +1,16 @@
+
+#pragma once
+
+#include <heap/seadDisposer.h>
+
+class DebugInput {
+    SEAD_SINGLETON_DISPOSER(DebugInput);
+
+public:
+    DebugInput() = default;
+    virtual ~DebugInput() = default;
+
+private:
+    bool t1 = false;
```

Comment:
To keep things consistent and obvious, the convention we usually use for placeholder names is `_offset_in_lowercase_hex`, so this would be `bool _8;`

(You may be wondering why the bool is at offset 8 and not offset 0: that's because the virtual function table pointer ("vtable pointer") is at offset 0.)